Documentation/performance/JitOptimizerTodoAssessment.md

```diff
@@ -99,36 +122,52 @@ benchmarking story set up to measure the effect of such changes when they go in,
 we should do that.
 
 
-Async
------
+Low-Hanging Fruit
+-----------------
 
-We've made note of the prevalence of async/await in modern code (and particularly
-in web server code such as TechEmpower), and have some opportunities listed in
-[#7914](https://github.com/dotnet/coreclr/issues/7914).  Some sort of study of
-async peanut butter to find more opportunities is probably in order, but what
-would that look like?
+### Switch Lowering
 
+The MSIL `switch` instruction is actually encoded as a jump table, so (for
+better or worse) intelligent optimization of source-level switch statements
+largely falls to the MSIL generator (e.g. Roslyn), since encoding sparse
+switches as jump tables in MSIL would be impractical.  That said, when the MSIL
+has a switch of just a few cases (as in [#12868](https://github.com/dotnet/coreclr/issues/12868)),
+or just a few distinct cases that can be efficiently checked (as in [#12477](https://github.com/dotnet/coreclr/issues/12477)),
+the JIT needn't blindly emit these as jump tables in the native code.  Work is
+underway to address the latter case in [#12552](https://github.com/dotnet/coreclr/pull/12552).
 
-Address Mode Building
----------------------
 
-One opportunity that's frequently visible in asm dumps is that more address
-expressions could be folded into memory operands' address expressions.  This
-would likely give a measurable codesize win.  Needs some thought about where
-to run in phase list and how aggressive to be about e.g. analyzing across
-statements.
+### Write Barriers
 
+A number of suggestions have been made for having the JIT recognize certain
+patterns and emit specialized write barriers that avoid various overheads --
+see [#13006](https://github.com/dotnet/coreclr/issues/13006) and [#12812](https://github.com/dotnet/coreclr/issues/12812).
 
-If-Conversion (cmov formation)
-------------------------------
 
-This hits big in microbenchmarks where it hits.  There's some work in flight
-on this (see #7447 and #10861).
+### Byref-Exposed Store/Load Value Propagation
+
+There are a few tweaks to our value-numbering for byref-exposed loads and stores
+to share some of the machinery we use for heap loads and stores that would
+allow better propagation through byref-exposed locals and out parameters --
+see [#13457](https://github.com/dotnet/coreclr/issues/13457) and
+[#13458](https://github.com/dotnet/coreclr/issues/13458).
+
+Miscellaneous
+-------------
+
+### Value Number Conservativism
+
+We have some frustrating phase-ordering issues resulting from this, but the
+opt-repeat experiment indicated that they're not prevalent enough to merit
+pursuing changing this right now.  Also, using SSA def as the proxy for value
+number would require handling SSA renaming, so there's a big dependency chained
+to this.
+Maybe it's worth reconsidering the priority based on throughput?
```

Comment:
What's the throughput argument?  Do we have a rough t-shirt size based on one of our scenarios?